### PR TITLE
docs: update cli example code

### PR DESCRIPTION
**Pull request by martinkrung**
### What I did

Make example code work with click

### How I did it

added a few lines
### How to verify it

got i working local

`Usage: ape run scripts/test.py [OPTIONS] COMMAND [ARGS]...

Options:
  -v, --verbosity LVL  One of ERROR, WARNING, SUCCESS, INFO, or DEBUG
  -h, --help           Show this message and exit.

Commands:
  cmd-1
  cmd-2
  cmd-3
`
### Checklist

- [x] Documentation has been updated

### CODE REVIEW

File: docs/userguides/clis.md [hunk 1]
```diff
@@ -105,18 +105,27 @@ Additionally, specify `ecosystem`, `network`, or `provider` in your command func
 import click
 from ape.cli import ConnectedProviderCommand
 
+@click.group()
+def cli():
+    pass
+
 @click.command(cls=ConnectedProviderCommand)
```
Comment (fubuloubu) [inline docs/userguides/clis.md:112]:
```suggestion
 @cli.command(cls=ConnectedProviderCommand)
```

File: docs/userguides/clis.md [hunk 2]
```diff
@@ -105,18 +105,27 @@ Additionally, specify `ecosystem`, `network`, or `provider` in your command func
...
-def cmd(network, provider):
+def cmd_1(network, provider):
    click.echo(network.name)
    click.echo(provider.is_connected)  # True
 
 @click.command(cls=ConnectedProviderCommand)
```
Comment (fubuloubu) [inline docs/userguides/clis.md:117]:
```suggestion
 @cli.command(cls=ConnectedProviderCommand)
```

File: docs/userguides/clis.md [hunk 3]
```diff
@@ -105,18 +105,27 @@ Additionally, specify `ecosystem`, `network`, or `provider` in your command func
...
-def cmd(provider):
+def cmd_2(provider):
    click.echo(provider.is_connected)  # True
 
 @click.command(cls=ConnectedProviderCommand)
```
Comment (fubuloubu) [inline docs/userguides/clis.md:121]:
```suggestion
 @cli.command(cls=ConnectedProviderCommand)
```

File: docs/userguides/clis.md [hunk 4]
```diff
@@ -105,18 +105,27 @@ Additionally, specify `ecosystem`, `network`, or `provider` in your command func
...
-def cmd():
+def cmd_3():
    click.echo("Using params from ConnectedProviderCommand is optional")
+
+
+cli.add_command(cmd_1)
+cli.add_command(cmd_2)
+cli.add_command(cmd_3)
```
Comment (fubuloubu) [inline docs/userguides/clis.md:128]:
```suggestion
```

**Review by fubuloubu — APPROVED** [review]
Good catch! Click groups allow you to create commands using functional decorator syntax

File: docs/userguides/clis.md [hunk 5]
```diff
@@ -105,17 +105,21 @@ Additionally, specify `ecosystem`, `network`, or `provider` in your command func
 import click
 from ape.cli import ConnectedProviderCommand
 
-@click.command(cls=ConnectedProviderCommand)
```
Comment (antazoey) [inline docs/userguides/clis.md:108]: this example wasn't demonstrating Ape's scripting system but was rather showing how you can use our CLI tools (anywhere).

**Review by antazoey — APPROVED** [review]
this looks good to me! it's a better example regardless

**Pull request merged**